ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -949,3 +949,52 @@ FAN_SS_MSM:
   - FAN
   - MSM
 
+FAN_SS_DICM2X:
+  guid: "32703a5c-7cad-4eb6-9605-08e92f1d2cdd"
+  description: "Basic fan with start/stop and status with dual dampers monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - DICM2X
+
+FAN_SS_DICM:
+  guid: "812965c4-c9cf-4a1a-af34-225ff7de5159"
+  description: "Basic fan with start/stop and status with damper monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - DICM
+
+FAN_SS_VSC_INVOPWM:
+  guid: "1c48e087-c973-4907-8d6e-93417da15d5b"
+  description: "Basic fan with start/stop and status with inverter monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - INVOPWM
```

Comment:
Can you just use the `power` fields included in VSC? why the need for the specific inverter type?

Reply:
<img width="404" alt="fan with inv" src="https://github.com/user-attachments/assets/0e5e979b-e7ec-4f7b-a81d-0d963a658158">

Reply:
the fans uses a inverter to run as shown above

Reply:
thank you for the information. In the future please refrain from posting and Google building specifc information here as this is an open source repo. Please send screenshots over email instead to bos-gps@google.com.